lib/project_types/theme/commands/push.rb

```diff
@@ -90,6 +82,20 @@ def self.help
 
       private
 
+      def push_completion_handler(theme:, has_errors:)
+        if options.flags[:json]
+          puts(JSON.generate(theme: theme.to_h))
```

Comment:
@karreiro should I add a `warning` field here? Seems odd to leave it out but not sure what you wanted to do.